docs/src/.vuepress/config.js

```diff
@@ -155,6 +155,9 @@ module.exports = {
                 children: [{
                   text: 'Intro',
                   link: '/guides/smartweave/warp/readme.md'
+                }, {
+                  text: 'Deploying Contracts',
+                  link: '/guides/smartweave/warp/deploying-contracts.md'
                 }, {
                   text: 'Readstate',
```

Comment:
should be seperate words or pascal case imo like Read State, or ReadState

Reply:
Not really a part of this PR